traffic_ops/traffic_ops_golang/vendor/gopkg.in/DATA-DOG/go-sqlmock/LICENSE

```diff
@@ -0,0 +1,28 @@
+The three clause BSD license (http://en.wikipedia.org/wiki/BSD_licenses)
+
```

Comment:
This license is fine, but I don't think I saw an appropriate corresponding entry in the top-level LICENSE file that references this and makes people aware that we've got a non-Apache license here. That definitely has to happen, unless I missed it.